README.md

```diff
@@ -4,7 +4,7 @@ A running camera feed that only saves videos when you press a button. How many t
 
 ## Project Status
 
-As of 06/11/2017, this project is in the "working prototype" phase of development. It will work, but please be patient with any bugs you are certain to encounter and any holes you may find in this documentation.
+As of 07/28/2020 with "Raspbian GNU/Linux 10 (buster)" OS, this project is in the "working prototype" phase of development. It will work, but please be patient with any bugs you are certain to encounter and any holes you may find in this documentation.
```

Comment:
Lets just call it `Raspberry Pi OS` as that is what people will google.

Reply:
there are 3 places where I will change this